content/influxdb/cloud/query-data/optimize-queries.md

```diff
@@ -72,7 +72,7 @@ We're continually optimizing Flux and this list may not represent its current st
 ## Balance time range and data precision
 To ensure queries are performant, balance the time range and the precision of your data.
 For example, if you query data stored every second and request six months worth of data,
-results will include a minimum of ≈15.5 million points.
-Flux must store these points in memory to generate a response.
+results would include ≈15.5 million points per series.  Depending on the [cardinality](https://docs.influxdata.com/influxdb/cloud/reference/glossary/#series-cardinality) (series count returned after your `filter()` function/s) of the query, this can quickly become many billions of points.
```

Comment:
- Links should just be relative (remove the protocol and domain)
- Depending on the number of series returned after `filter()` ([cardinality]()), this can quickly become many billions of points.